gateway-js/src/__tests__/integration/configuration.test.ts

```diff
@@ -320,46 +317,6 @@ describe('gateway config / env behavior', () => {
       );
     });
   });
-
-  describe('schema config delivery endpoint configuration', () => {
```

Comment:
**note** These were moved to be part of integration tests, since they really should be tested as part of gateway construction

https://github.com/apollographql/federation/pull/1812/files#diff-9351bdb42a25121eac9a7bf90afbca18146a69c980ed56acbd158bcda26b80b7R88-R107